sdk/dell/arm-dell-storage/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
     
-## 1.0.0-beta.1 (2025-07-18)
+## 1.0.0-beta.1 (2025-12-29)
```

Comment:
why still  beta?
@skywing918  is this because we have not release a new version for our release tool after https://github.com/Azure/azure-sdk-tools/pull/13094 merged?